src/styles/blocks/header.scss

```diff
@@ -0,0 +1,187 @@
+.header {
+  position: relative;
+
+  @include onTablet {
+    height: 384px;
+  }
+
+  @include onDesktop {
+    height: 655px;
+  }
+
+  &::after {
+    content: "";
+
+    position: absolute;
+    z-index: -1;
+    top: 0;
+    left: 0;
+    right: 0;
+
+    background-image: url(../images/photos/header.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    height: 300px;
+
+    @include onTablet {
+      height: 384px;
+      left: 45%;
+    }
+
+    @include onDesktop {
+      height: 100%;
+      left: 45%;
+    }
+
+  }
+
+  &__menu-opener {
+    background-image: url(../images/menu.svg);
+    width: 18px;
+    height: 12px;
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__nav {
+    display: flex;
+    align-items: center;
+    padding-top: 9px;
+    margin-bottom: 275px;
```

Comment:
Consider using variables or calculations for consistent spacing and margins throughout the code.